packages/framework/src/Model/Feed/FeedExportFactory.php

```diff
@@ -31,16 +33,30 @@ class FeedExportFactory
      */
     protected $feedPathProvider;
 
+    /**
+     * @var \Symfony\Component\Filesystem\Filesystem
+     */
+    private $localFilesystem;
+
+    /**
+     * @var \League\Flysystem\MountManager
+     */
+    private $mountManager;
+
     public function __construct(
```

Comment:
does it make sense to create annotations for constructors like it is in document about decision about annotations?